lib/public/views/Logs/Details/index.js

```diff
@@ -48,15 +47,18 @@ const logDetailScreen = (model) => {
             return items;
         };
 
-        return h('', h('.w-100.flex-column', tree(data[0])));
-    } else if (error) {
         return h('', [
-            h('.danger', 'This log could not be found.'),
-            h('button.btn.btn-primary.mv3', {
+            h('h2.mv2', { onremove: () => model.logs.clearLogs() }, 'Log Tree'),
+            h('.w-100.flex-column', tree(data.payload[0])),
+        ]);
+    } else if (data.isFailure()) {
+        return h('', [
+            h('.alert.alert-danger', 'This log could not be found.'),
```

Comment:
We are not sure what the errors is, so we should use something like:
```js
data.payload.map((error) => h('.alert.alert-danger', h('b', `${error.title}: `), error.detail))
```

Reply:
Done